.github/workflows/e2e_additional_editor_and_player.yml

```diff
@@ -54,3 +54,5 @@ jobs:
         with:
           name: protractor-video
           path: /home/runner/work/oppia/protractor-video
+      - name: Checking Memory Usage
```

Comment:
is it worth of checking memory usage after the completion of test or after shutting down the server?

Reply:
I asked about this with @U8NWXD  . He told that the statement gives the output of the maximum memory used in the CI job run.

Reply:
Ah, I see! I misunderstood it earlier. Thanks for the explanation!